operator/deploy/role.yaml

```diff
@@ -223,6 +223,16 @@ rules:
   - routes
   verbs:
   - get
+- apiGroups: ["console.openshift.io"]
```

Comment:
also need to update the golden copies of the manifests (the OLM stuff). See the `openshift/manifests` directory. Need to prepare new versions with this new stuff.

Reply:
I just noticed that these are permissions for the operator itself not kiali, so we need to move these up above the comment:

`# The permissions below are for Kiali itself; operator needs these so it can escalate when creating Kiali's role`

I'm going to submit a PR to this PR to introduce a molecule test -- I'll submit a change for this as well as changes to the gold copies of the manifests.